internal/services/storage/storage_account_resource_test.go

```diff
@@ -1138,6 +1158,14 @@ func TestAccStorageAccount_infrastructureEncryptionStorageV2(t *testing.T) {
 			),
 		},
 		data.ImportStep(),
+	})
+}
+
+func TestAccStorageAccount_infrastructureEncryptionStorageV2_Disabled(t *testing.T) {
```

Comment:
splitting this into a separate test means we won't be testing the update path here?

Reply:
This (and the others) are split since the interested property is a force new one, it is meaningless to test for the update scenario anyway.